### PR TITLE
Small update for zuse library changes.

### PR DESCRIPTION
**Pull request by ngzax**
## Why?
I made this update to @Fang- 's library as well, but it must have been after it was copied to examples.

### CODE REVIEW

File: app/string-tester.hoon [hunk 1]
```diff
@@ -1,12 +1,8 @@
...
 /+  string
 !:
-|%
-++  move  {bone card}
-++  card  $%  ~
-          ==
---
-|_  {bowl ~}
+|_  {bowl:gall $~}
+
```
Comment (Fang-) [inline app/string-tester.hoon]: While this line *is* in my repo's version, it really shouldn't be there. Whitelines begone!

**Review by Fang- — CHANGES_REQUESTED** [review]
I don't like how we have so much old and outdated code in the examples. Thanks for at least making this compile! One little nit in here, I'll merge after that's fixed. (:

File: app/string-tester.hoon (hunk 1 above)
Comment (ngzax) [reply app/string-tester.hoon]: Ok, I'll fix it in both. 😄  I was going to send a small Readme update for that repo as well.

**Review by Fang- — APPROVED** [review]
Amazing, thanks. (:

**Pull request merged**